website/snippets/metadata-api-prerequisites.md

```diff
@@ -0,0 +1,5 @@
+## Prerequisites
+
+- You must have a [multi-tenant](/docs/deploy/regions-ip-addresses) account 
+- You must be on a [Team or Enterprise plan](https://www.getdbt.com/pricing/)
+- Your projects must be on dbt version 0.19.0 or higher. Refer to [Version migration guides](/guides/migration/versions) to upgrade
```

Comment:
@cafzal - wanted to double check if this is correct given we've deprecated versions <1.0? i think this should be 1.0 and higher right?

Reply:
Good catch, that's correct @mirnawong1